packages/components/tree-v2/src/emits.ts

```diff
@@ -0,0 +1,17 @@
+export const NODE_CLICK = 'node-click'
+export const NODE_EXPAND = 'node-expand'
+export const NODE_COLLAPSE = 'node-collapse'
+export const CURRENT_CHANGE = 'current-change'
+export const NODE_CHECK = 'check'
+export const NODE_CHECK_CHANGE = 'check-change'
+export const NODE_CONTEXTMENU = 'node-contextmenu'
+
+export const TREE_EMITS = [
+  NODE_CLICK,
+  NODE_EXPAND,
+  NODE_COLLAPSE,
+  CURRENT_CHANGE,
+  NODE_CHECK,
+  NODE_CHECK_CHANGE,
+  NODE_CONTEXTMENU,
+]
```

Comment:
I think we can define these into the same file for `defaults` and `emits` in your case `virtual-tree.ts` 

Ref: https://github.com/element-plus/element-plus/blob/dev/packages/components/affix/src/affix.ts